### PR TITLE
Update data-sources.md

### PR DESCRIPTION
**Pull request by ssaldner**
After discussing this with Laura Portell Silva, I transferred the contents from the old draft to the new template. Some sections had quite limited content, and were therefore merged with others. I also made a few additions and edits where necessary. The Google Docs version, including more comments and explanation can be found [here](https://docs.google.com/document/d/1GAljkcNYJqbJcrfvv7U00arj5seWpc2xYsuivltTy7Q/edit).

Best wishes,
/Simon Saldner - DANS-KNAW - simon.saldner@dans.knaw.nl

### CODE REVIEW

File: socioeconomic-data/data-sources.md [hunk 1]
```diff
@@ -29,81 +29,47 @@ New content is announced on the [home page](/) and [news page](/about/news), so
...
+- **Sensitive data** and privacy considerations: due to the prevalence of sensitive and personal data in the domain of socioeconomic research, access to such data is often limited by serious data protection and privacy concerns, often making the use and sharing of such data difficult. Solutions such as pseudonimising data, or providing access via secure environments such as the [ODDISEI Secure ANalysis Environment (SANE)](https://odissei-data.nl/en/2022/02/secure-analysis-environment-sane-secure-data-for-social-sciences-and-humanities/) can often provide some level of access to sensite data, yet much of this data is only available on request, or as aggregated data.
```
Comment (lauportell) [inline socioeconomic-data/data-sources.md:45]:
```suggestion
- **Sensitive data and privacy**: due to the prevalence of sensitive and personal data in the domain of socioeconomic research, access to such data is often limited by serious data protection and privacy concerns, often making the use and sharing of such data difficult. Solutions such as pseudonymising data, or providing access via secure environments such as the [ODDISEI Secure ANalysis Environment (SANE)](https://odissei-data.nl/en/2022/02/secure-analysis-environment-sane-secure-data-for-social-sciences-and-humanities/) can often provide some level of access to sensitive data, yet much of this data is only available on request, or as aggregated data. 
```

File: socioeconomic-data/data-sources.md [hunk 2]
```diff
@@ -29,81 +29,47 @@ New content is announced on the [home page](/) and [news page](/about/news), so
...
 
-<!--- Example of topic, optional --->
+From the demography perspective, there have been surveys and related publications proving the relation of household structures, housing conditions (Giorgi & Boertien 2021), the role of nursing homes, especially during the first wave (Bernandi et all 2021), the role of gender or socioeconomic status (Horton 2021) etc. From the perspective of political science, psychology or sociology sciences’  perspective, surveys that studied the effect of conspiracy theories regarding vaccines, propagation of virus and others, have show clusters of people that relate resistance to vaccines with respect to personal ideology , adherence to conspiracy theories or that lower level of education, religiosity, conspiracy thinking can be an indicator for vaccine resistance (Haaksonen & Furman 2022).
```
Comment (lauportell) [inline socioeconomic-data/data-sources.md:53]: I think we need to find a way to add these references as it has to be consistent from other pages.
Could you find a DOI for all of them so we can add them maybe?

File: socioeconomic-data/data-sources.md [hunk 3]
```diff
@@ -29,81 +29,47 @@ New content is announced on the [home page](/) and [news page](/about/news), so
...
 
-<!--- Example of topic, optional --->
+BY-COVID uses two primary, and multiple secondary data sources for socioeconomic data, which will be outlined in this section. Both primary and secondary data sources provide data that relate to or are relevant for COVID-19 and other infectious disease outbreaks, and therefore the BY-COVID project. The two primary data were determined early on the project, while the list of secondary sources will be progressively identified throughout the course of the project. These data sources are filtered for data relevant to the project, and the resulting metadata is prepared and harmonized via a [harvesting tool](https://docs.google.com/document/d/1a5mbLb3rNVTofD4jTTeIqN3qJnb2QhBI/edit#) before being added to the COVID-19 Data Platform (for more details, see project deliverable [D2.1](https://docs.google.com/document/d/1a5mbLb3rNVTofD4jTTeIqN3qJnb2QhBI/edit#heading=h.7ngyrwsprxyn)).
```
Comment (lauportell) [inline socioeconomic-data/data-sources.md:65]:
```suggestion
BY-COVID uses two primary, and multiple secondary data sources for socioeconomic data, which will be outlined in this section. Both primary and secondary data sources provide data that relate to or are relevant for COVID-19 and other infectious disease outbreaks, and therefore the BY-COVID project. The two primary data were determined early on the project, while the list of secondary sources will be progressively identified throughout the course of the project. These data sources are filtered for data relevant to the project, and the resulting metadata is prepared and harmonized via a [harvesting tool](https://t2-4.by-covid.bsc.es/jspui/) before being added to the COVID-19 Data Platform (for more details, see project deliverable [D2.1](https://zenodo.org/record/7017728).
```
Changed the links to the webpage of the harvester and the zenodo DOI of the deliverable

File: socioeconomic-data/data-sources.md [hunk 4]
```diff
@@ -29,81 +29,47 @@ New content is announced on the [home page](/) and [news page](/about/news), so
...
 
-Under what licenses do the data sources work?
 
 ### Considerations
 
-Using a bullet point style list format as much as possible, describe the considerations for your topic that are specific to infectious diseases within the category (e.g. pathogen characterisation) that you have selected.
-
-Please avoid replicating 'generic' guidelines, i.e. those not specific to infectious diseases, here. Add links to RDMkit in the metadata above, if any are needed. 
+- **Licenses**: Socio-economic data sources are generally public and accessible, but authorization or licensing agreements may be required to access some datasets. While the complete datasets may not always be openly available, the metadata is typically freely accessible. Socio-economic data sources in the BY-COVID project also offer programmatic access to their metadata using standards like the Open Archives Initiative Protocol for Metadata Harvesting (OAI-PMH). This enables the automated harvesting, harmonisation, and transformation of metadata into the OmicsDI format, which can then be utilised by the Covid-19 Data Portal.
```
Comment (lauportell) [inline socioeconomic-data/data-sources.md:70]:
```suggestion
- **Licenses**: Socio-economic data sources are generally public and accessible, but authorization or licensing agreements may be required to access some datasets. While the complete datasets may not always be openly available, the metadata is typically freely accessible. Socio-economic data sources in the BY-COVID project also offer programmatic access to their metadata using standards like the Open Archives Initiative Protocol for Metadata Harvesting (OAI-PMH). This enables the automated harvesting, harmonisation, and transformation of metadata into the OmicsDI format, which can then be used by the COVID-19 Data Portal.
```
Changed Covid-19 Data Portal to COVID-19 Data Portal for consistency

**Review by lauportell — CHANGES_REQUESTED** [review]
iguyguyguyg

**Comment by lauportell** [comment]
@ssaldner I think it is good and only small changes have to be done.
I have also added the tools related to this page to the [tools table](https://docs.google.com/spreadsheets/d/13tqfSbgivokfEkxGXPRFShVhCmO4VuTQRe4uQgJOMbc/edit#gid=0).
Can you review that they are correct? I have added:
- COVID-19 Data Portal
- ODDISEI Secure ANalysis Environment (SANE)
- CESSDA Data Catalogue (CDC)
- EUI COVID-19 SSH Data Portal
Thank you!

File: socioeconomic-data/data-sources.md (hunk 2 above)
Comment (ssaldner) [inline socioeconomic-data/data-sources.md:53]:
```suggestion
From the demography perspective, there have been surveys and related publications proving the relation of household structures, housing conditions ([Giorgi & Boertien 2021: DOI: 10.1186/s41118-021-00124-8](https://doi.org/10.1186/s41118-021-00124-8)), the role of nursing homes, especially during the first wave ([Bernandi et al. 2021. DOI: 10.1186/s41118-021-00119-5] (https://doi.org/10.1186/s41118-021-00119-5)), the role of gender or socioeconomic status [Horton 2020. DOI: 10.1016/S0140-6736(20)32000-6](https://pubmed.ncbi.nlm.nih.gov/32979964/) etc. From the perspective of political science, psychology or sociology sciences’  perspective, surveys that studied the effect of conspiracy theories regarding vaccines, propagation of virus and others, have show clusters of people that relate resistance to vaccines with respect to personal ideology , adherence to conspiracy theories or that lower level of education, religiosity, conspiracy thinking can be an indicator for vaccine resistance (Haaksonen & Furman 2022). 
```

**Review by ssaldner — COMMENTED** [review]
Hi Laura, thanks again for reviewing. As said, I accepted most of your changes, and added some new references in the text

**Review by lauportell — APPROVED** [review]

**Pull request merged**